src/screens/BlockUser/BlockUser.test.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { act, render, screen } from '@testing-library/react';
+import { MockedProvider } from '@apollo/react-testing';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import 'jest-location-mock';
+
+import BlockUser from './BlockUser';
+import { USER_LIST } from 'GraphQl/Queries/Queries';
+import {
+  BLOCK_USER_MUTATION,
+  UNBLOCK_USER_MUTATION,
+} from 'GraphQl/Mutations/mutations';
+import { store } from 'state/store';
+import userEvent from '@testing-library/user-event';
+
+const MOCKS = [
+  {
+    request: {
+      query: USER_LIST,
+    },
+    result: {
+      data: {
+        users: [
+          {
```

Comment:
why dummy data

Reply:
This dummy data is to mock graphql queries used in BlockUser.tsx file.